curvefs/src/metaserver/storage/converter.cpp

```diff
@@ -154,15 +150,10 @@ Key4S3ChunkInfoList::Key4S3ChunkInfoList(uint32_t fsId,
       lastChunkId(lastChunkId),
       size(size) {}
 
-std::string Key4S3ChunkInfoList::SerializeToString() const {
-    std::ostringstream oss;
-    oss << keyType_ << ":" << fsId << ":"
-        << inodeId << ":" << chunkIndex << ":"
-        << std::setw(kMaxUint64Length_) << std::setfill('0') << firstChunkId
-        << ":"
-        << std::setw(kMaxUint64Length_) << std::setfill('0') << lastChunkId
-        << ":" << size;
-    return oss.str();
+inline std::string Key4S3ChunkInfoList::SerializeToString() const {
+    return absl::StrCat(keyType_, ":", fsId, ":", inodeId, ":",
+        chunkIndex, ":", absl::StrFormat("%020lu", firstChunkId), ":",
+        absl::StrFormat("%020lu", lastChunkId), ":", size);
```

Comment:
should we use `PRIu64` instead of `lu` in the format specification?

http://jhshi.me/2014/07/11/print-uint64-t-properly-in-c/index.html#.YvZdUHZBxPY

Reply:
fix.